server/server.js

```diff
@@ -100,4 +98,15 @@ app.get('/tasks', (req, res) =>
         })
 );
 
+app.post('/task/updated', (req, res) => {
```

Comment:
Looks better! You can improve it by using the promise-based (compared to the current callback-based) version of this method.

```
Task.findOneAndUpdate({ _id: req.body._id }, { $set: { status: req.body.status } })
	.then(task => {...})
	.catch(error => {...});
```

Return the `Task.findOneAndUpdate` call from the request handler (exactly like in the GET /tasks)